python/tests/expressions/test_visitors.py

```diff
@@ -874,36 +881,19 @@ def test_manifest_evaluator_less_than_overlap():
             upper_bound=_to_byte_buffer(StringType(), "b"),
         )
     )
+    assert not _ManifestEvalVisitor(string_schema, expr).eval(manifest)
 
-    assert not _create_manifest_evaluator(expr).eval(manifest)
-
-
-def test_manifest_evaluator_less_than_all_null():
-    expr = BoundLessThan(
-        term=BoundReference(
-            field=NestedField(field_id=1, name="foo", field_type=StringType(), required=False),
-            accessor=Accessor(position=0, inner=None),
-        ),
-        literal=StringLiteral("a"),
-    )
 
+def test_manifest_evaluator_less_than_all_null(string_schema: Schema):
+    expr = LessThan[str](Reference("col_str"), StringLiteral("a"))
     manifest = _to_manifest_file(
         PartitionFieldSummary(contains_null=False, contains_nan=False, lower_bound=None, upper_bound=None)
```

Comment:
This doesn't look right. If all values are null then `contains_null` should be `True`.